argocd/overlays/moc-infra/applications/envs/moc/smaug/kruize/kruize.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: hpo
+spec:
+  destination:
+    name: smaug
+    namespace: openshift-tuning
```

Comment:
This namespace is not allowed in the `kruize` ArgoCD project, see:

https://github.com/operate-first/apps/blob/4b724a89068ca34cafac695d542516f5d4b5cfc7/argocd/overlays/moc-infra/projects/kruize.yaml#L9


(`openshift-tuning-*` doesn't match `openshift-tuning` because of the `-*`)

Please add the additional namespace to the project.